src/connectors/doorhanger/doorhanger.js

```diff
@@ -24,10 +26,10 @@ const doorhangerStyle = css`
     top: 10px;
     right: 10px;
     transform: translateY(-150%);
-    display: flex;
-    justify-content: flex-start;
-    align-items: center;
-    align-content: center;
+    /* display: flex; */
+    /* justify-content: flex-start; */
+    /* align-items: center; */
+    /* align-content: center; */
```

Comment:
This may end up uncommented, but for now it was causing all sorts of display issues

Reply:
And these were the culprit styles preventing it from displaying on other pages.

Reply:
🙌 Thank you for digging into this one! It was driving me up the wall!